install.sh

```diff
@@ -55,6 +55,21 @@ then
               #rpm --import https://www.elrepo.org/RPM-GPG-KEY-elrepo.org
               #rpm -Uvh http://www.elrepo.org/elrepo-release-7.0-2.el7.elrepo.noarch.rpm
               #yum --enablerepo=elrepo-kernel -y install kernel-ml
+              
+              # AUTO-CONFIGURE MOUNT OPTIONS
+              echo "Auto-Configuring mount options..."
+
+              # Stores name of partitions(space seperated) on which caches are present
+              cache_prt=$(mount | grep -E '/\<cache\> | /cache[[:digit:]]+' | awk '{print $1}')
+              echo "Cache partitions are :"
+              echo $(tput bold)$cache_prt $(tput sgr0)
+              
+              # escaping the "/" with "\/"
+              cache_prt=$(echo $cache_prt | sed 's:/:\\/:g')
```

Comment:
Now its ok, but there is still a problem, the fstab file contains the uuid of the partition instead of the partition name
say UUID=4adbfe16-6277-4b19-b976-e11a5b4bce69 /boot
Why not use /cache1 directly to change instead of the actual partition that contains it? Am I missing something here?

Reply:
Hmm, nice suggestion, it can be done, Initially I thought that it would be a fail safe type of thing, to only edit entries which are present in the output of mount. Will fix it, I think its now only a one liner.😂